frontend/utils/regexp.ts

```diff
@@ -3,3 +3,5 @@ export const REGEXP_HAS_NUMBERS = /^(?!0+$)\d+$/
 export const REGEXP_VALID_EMAIL = /[a-z0-9!#$%&'*+/=?^_`{|}~-]+(?:\.[a-z0-9!#$%&'*+/=?^_`{|}~-]+)*@(?:[a-z0-9](?:[a-z0-9-]*[a-z0-9])?\.)+[a-z0-9](?:[a-z0-9-]*[a-z0-9])?/i
 
 export const REGEXP_VALID_NAME = /^(?=.*[a-zA-Z0-9])[a-zA-Z0-9 ]*$/
+
+export const REGEXP_PUBLIC_KEX = /^(1|3)[A-Za-z0-9]{39}$/
```

Comment:
I am not an expert of regular expressions.
I believe that this one is not what we want and I asked ChatGPT 4o as a second opinion.
I think that we want
`/^(|0x)[A-Fa-f0-9]{96}$/`

Reply:
your AI is better then my AI ;-)